yarn.lock

```diff
@@ -2,7 +2,7 @@
 # yarn lockfile v1
 
 
-"@babel/code-frame@7.0.0-beta.44", "@babel/code-frame@^7.0.0-beta.40":
```

Comment:
it's not normal that there are so many changes in the `yarn.lock` if you just updated `prettier`.